src/components/atoms/Logo/Logo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { DEFAULT_VENUE_LOGO_PNG } from "settings";
+
+interface LogoProps {
+  width?: string;
+  height?: string;
+}
+
+export const Logo: React.FC<LogoProps> = ({
+  width = "100%",
+  height = "100%",
+}) => {
+  return (
+    <img
+      src={DEFAULT_VENUE_LOGO_PNG}
+      width={width}
+      height={height}
+      alt="Sparkle Logo"
+    />
+  );
+};
```

Comment:
We already have this component
![image](https://user-images.githubusercontent.com/63194656/127016358-e7961403-0a76-4f51-bf0e-66d4914e7d80.png)